src/api/app/models/bs_request_action_set_bugowner.rb

```diff
@@ -1,47 +1,18 @@
 #
 class BsRequestActionSetBugowner < BsRequestAction
-  #### Includes and extends
-  #### Constants
-
-  #### Self config
   def self.sti_name
     :set_bugowner
   end
 
-  #### Attributes
-  #### Associations macros (Belongs to, Has one, Has many)
-  #### Callbacks macros: before_save, after_save, etc.
-  #### Scopes (first the default_scope macro if is used)
-  #### Validations macros
-  #### Class methods using self. (public and then private)
-  #### To define class methods as private use private_class_method
-  #### private
-
-  #### Instance methods (public and then protected/private)
-  def check_sanity
-    super
-    return unless person_name.blank? && group_name.blank?
-
-    errors.add(:person_name, 'Either person or group needs to be set')
+  def role
+    'set_bugowner'
```

Comment:
I would really prefer to call this `'bugowner'`, but then I have the problem that I need the type and the role in [the `creator_intentions` helper](https://github.com/openSUSE/open-build-service/pull/4748/files#diff-d6b551dedaaae6cce3a56f2afb75ef6fR369), and the parameters depend on each other. @ChrisBr was completely against this in https://github.com/openSUSE/open-build-service/pull/4714#discussion_r176504794. He suggested there:

``` ruby
 role == :bugowner ? 'become bugowner' : "get the role #{role}"
```

but then both `set_bugowner` and `add_role bugowner` would have role `bugowner`, and they are different things and need to be rendered differently. 

Calling this `set_bugowner` instead of `bugowner` just to write simpler a helper... :sob: